hyrisecockpit/frontend/src/services/transformationService.ts

```diff
@@ -36,6 +38,19 @@ export function useDataTransformation(metric: Metric): TransformationService {
 
 // TRANSFORM FUNCTION POOL
 
+/** transform query information data to table structure */
+function getQueryInformationData(data: any, primaryKey: string = ""): any {
+  const entry = data.find((dbEntry: any) => dbEntry.id === primaryKey);
+  return entry.query_information.map((query: any) => {
```

Comment:
```suggestion
  return entry.detailed_query_information.map((query: any) => {
```
This was changed in a backend PR (see #722). It also needs to be adapted in the test code. I am going to do this. This comment is just a TODO for me